### ISSUE TITLE
Dragging an issue to the top of the column sometimes causes it to dissappear

### ISSUE THREAD

**Issue by adamreisnz** [ZenHubIO/support#1217]
See the following situation where it happened:

![image](https://user-images.githubusercontent.com/490562/29435738-4d1ebebc-83fc-11e7-8751-f761469290c3.png)

The missing issue is in fact still visible, it's the 2px grey line visible just above the other issue. When I click that grey line, the modal with the missing issue pops up:

![image](https://user-images.githubusercontent.com/490562/29435778-75bed4a6-83fc-11e7-8158-7c1008192baa.png)

Here's the HTML structure for debugging purposes:

```html
<div class="zhc-issue-cards__cell" style="height: 7px; left: 0px; position: absolute; top: 0px; width: 255px;"><div class="zhc-issue-card" draggable="true"><div class="zhc-issue-card__header"><div class="zhc-issue-card__section"><div class="zhc-issue-card__section__meta"><div class="zhc-tooltip-wrapper"><div class="zhc-issue-card__assignee"><div class="zhc-avatar " style="width: 25px; height: 25px; background-image: none;"><img src="https://avatars0.githubusercontent.com/u/490562?v=4&amp;size=50" alt="adamreisnz avatar"></div><div class="zhc-issue-card__assignees zhc-issue-card__assignees--second"><div class="zhc-avatar " style="width: 15px; height: 15px; background-image: none;"><img src="https://avatars1.githubusercontent.com/u/20091343?v=4&amp;size=30" alt="peterfeldbrugge avatar"></div></div></div></div></div><div class="zhc-issue-card__section__content"><div class="zhc-issue-card__heading"><svg class="zhc-issue-card__select-icon" width="16" height="16" viewBox="0 0 16 16"><circle fill="#9b9b9b" stroke="#fff" cx="8" cy="8" r="7"></circle><path fill="none" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" d="M4.737,8.283l2.205,2.18l4.32-4.926"></path></svg><div class="zhc-issue-card__select-message">Click to select</div><div class="zhc-issue-card__heading__main"><!-- react-empty: 671 --><span class="zhc-issue-card__repo-name">jolinejolink</span><span class="zhc-issue-card__issue-number"><!-- react-text: 674 -->#<!-- /react-text --><!-- react-text: 675 -->416<!-- /react-text --></span><!-- react-empty: 676 --><div class="zhc-issue-card__actions"><div class="zhc-tooltip-action-wrapper"><button class="zhc-tooltip-action-wrapper__button"><span class="zhc-icon zhc-icon--gear zhc-icon--grey-light zhc-icon--highlight-hover"></span></button></div></div></div></div><div class="zhc-issue-card__issue-title">zoekfunctie site optimaliseren</div></div></div></div><div class="zhc-issue-card__main"><!-- react-empty: 683 --><!-- react-empty: 684 --><!-- react-empty: 685 --><!-- react-empty: 686 --></div><div class="zhc-issue-card__meta"><div class="zhc-issue-card__meta__badges"><span class="zhc-badge zhc-badge--estimate"><span class="zhc-badge__value">3</span></span><span class="zhc-label" style="background-color: rgb(32, 125, 229); color: rgb(255, 255, 255);">enhancement</span></div></div></div></div>
```

Looks like the content is all there, but the card container was assigned a fixed height of 7px for some reason.

There is also the following warning in the console, but not sure if that has to do with it:

![image](https://user-images.githubusercontent.com/490562/29435826-a8ab8788-83fc-11e7-9ec2-8e904e9531b1.png)

Note that this has happened more than once before.

**Comment by azenMatt** [comment]
Thanks for the report @adamreisnz 

we will take a look into this. We are also currently looking at improving the drag and drop 'space' at the top and bottom of a pipeline. Sometimes it can be a real art to hit the 'sweet-spot' that causes the pipeline to move. These may be related

**Comment by adamreisnz** [comment]
Just happened again, seems to occur when dragging an issue to a previously empty column.

**Comment by adamreisnz** [comment]
It's not just to an empty column or top of the column. I dragged an issue to between two others and it also disappeared, becoming a thin line:

![image](https://user-images.githubusercontent.com/490562/29947469-f09b3f3c-8efd-11e7-9f7d-3cf4c017ec43.png)

**Comment by azenMatt** [comment]
Hi @adamreisnz - we have pushed out a number of updates to drag-and-drop over this week, including a fix which should handle this problem. 

unfortunately, I haven't been unable to reliably recreate the issue pre-fix, so I can't say with 100% conviction that we have it. 

I'l leave the issue open for the moment if you could let me know if you see it reoccur? if neither of us see it for a while, i will close this issue, thanks

**Comment by adamreisnz** [comment]
Excellent thanks, will do. Yes I could never reliably reproduce, but it happened frequently enough, at least a few times a week. So I'll give it a week or so and get back to you.

**Comment by adamreisnz** [comment]
@azenMatt Unfortunately this has happened again. This time not in an empty column, and not at the top of the column, but between two other issues.

What happened is I tried to drag the card between two cards, but then the empty space that is the placeholder when hovering disappeared/shrunk, and the existing cards in the column shifted up. When I released the mouse and dropped the card, it had shrunk into a line between two cards as visible here:

![image](https://user-images.githubusercontent.com/490562/31049733-b06d2eac-a695-11e7-8dd7-20ece06176bf.png)

So basically this is exactly the same thing as used to happen before.
